kustomization/components/kube-vip/rbac.yml

```diff
@@ -23,9 +23,9 @@ rules:
       - endpoints
       - services
     verbs:
+      - endpoints
       - get
       - list
-      - update
```

Comment:
This turns out to be problematic.  I added a new service that is of type `LoadBalancer` and added the annotation, and it cannot update it:
```
2025/05/12 02:50:55 ERROR updating Spec service=loki-distributor err="services \"loki-distributor\" is forbidden: User \"system:serviceaccount:kube-vip:kube-vip\" cannot update resource \"services\" in API group \"\" in the namespace \"monitoring\""
```

Reply:
Ah, this was fixed upstream: https://github.com/kube-vip/kube-vip/pull/1146

In the mean time, I'm just going to patch this without codegen.

Reply:
Fixed in https://github.com/marinatedconcrete/config/commit/e31b9b9705dcfc58df5ece80a6b3ebdad4be9f6b